plugins/main/public/controllers/agent/components/agents-table.js

```diff
@@ -252,15 +252,14 @@ export const AgentsTable = withErrorBoundary(
         const rawAgents = await this.props.wzReq('GET', '/agents', {
           params: { ...this.buildFilter(), select: selectFields },
         });
-        const formatedAgents = (
-          ((rawAgents || {}).data || {}).data || {}
-        ).affected_items.map(this.formatAgent.bind(this));
+        const formatedAgents = rawAgents?.data?.data?.affected_items.map(
```

Comment:
Question:
In this line if rawAgents?.data?.data?.affected_items is undefined, then undefined.map throws an error. Right?

Reply:
Done, thanks for letting me know!